smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ClientRuntimeTypes.kt

```diff
@@ -88,8 +88,10 @@ object ClientRuntimeTypes {
     }
 }
 
-private fun runtimeSymbol(name: String): Symbol = buildSymbol {
+private fun runtimeSymbol(name: String, defaultValue: String? = null, nullable: Boolean = true): Symbol = buildSymbol {
     this.name = name
     this.namespace = SwiftDependency.CLIENT_RUNTIME.target
+    this.defaultValue = defaultValue
+    this.nullable = nullable
```

Comment:
1.  Where is this new field `nullable` being used?
2. Probably will get cleared up once we answer #1, but, can't everything be nullable?

Reply:
1. https://github.com/awslabs/smithy-swift/blob/main/smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/model/SymbolBuilder.kt#L72
2. technically yes but per spec no. actually we might not need this. I might have messed up. checking one thing.